rust/src/lib/nispor/route_rule.rs

```diff
@@ -83,6 +83,8 @@ pub(crate) fn get_route_rules(
         };
         rules.push(rule);
     }
+
+    rules.sort_unstable_by_key(|rule| rule.priority.unwrap_or(0));
```

Comment:
When routing rule priority is not specified by the user, we can not just blindly assign 0 to the rule for sorting purpose, if you do `ip -4 rule show`, you would notice that rule lookup local table has priority 0.   Instead, we should prefer the rule with priority specified during sorting, and the lower the better. And those rules without the priority specified should be sorted into last row during the `nmstatectl show`. @cathay4t , correct me if my idea is wrong.     

```
$ ip -4 rule
0:	from all lookup local
32766:	from all lookup main
32767:	from all lookup default
```

Reply:
This PR is for rule from nispor where kernel always has priority set.